src/app/shared/pipes/safe.pipe.ts

```diff
@@ -16,25 +16,81 @@ export class SafePipe implements PipeTransform {
         if (fromEmail === 'support@ctemplar.com') {
           return this.sanitizer.bypassSecurityTrustHtml(value);
         }
+        const cssFilter = new cssfilter.FilterCSS({
+          onIgnoreAttr: (styleName, styleValue, opts) => {
+            const blackList = {
+              position: ['fixed']
+            };
+            if (blackList.hasOwnProperty(styleName)) {
+              const blackValList = blackList[styleName];
+              const val = styleValue.replace(/!important/g, '').trim();
+              if (blackValList.includes(val)) {
+                return '';
+              }
+            }
+            return styleName + ':' + xss.friendlyAttrValue(styleValue);
+          }
+        });
         const xssValue = xss(value, {
           stripIgnoreTag: true,
           stripIgnoreTagBody: ['script', 'style'],
-          onIgnoreTagAttr: (tag, attrName, attrValue, isWhiteAttr) => {
-            const safeAttrValue = xss.safeAttrValue(tag, attrName, attrValue, new cssfilter.FilterCSS({
-              onIgnoreAttr: (styleName, styleValue, opts) => {
-                const blackList = {
-                  position: ['fixed']
-                };
-                if (blackList.hasOwnProperty(styleName)) {
-                  const blackValList = blackList[styleName];
-                  const val = styleValue.replace(/!important/g, '').trim();
-                  if (blackValList.includes(val)) {
-                    return '';
+          onTag: (tag, html, options) => {
+            if (tag === 'a' && !(options && options['isClosing'] === true)) {
+              let htmlAttrs = '';
+
+              // "<a href='' target=''>" => "href='' target=''"
+              const reg = /\s|\n|\t/;
+              const match = reg.exec(html);
+              const i = match ? match.index : -1;
+              if (i !== -1) {
+                htmlAttrs = html.slice(i + 1, -1).trim();
+              }
+
+              let containsTargetAttr = false;
+              let attrsHtml = xss.parseAttr(htmlAttrs, (attrName, attrValue) => {
+                if (attrName === 'target') {
+                  containsTargetAttr = true;
+                  return attrName + '="_new"';
```

Comment:
@angelo510 it should be `target="_blank"` and not `target="_new"`.
https://stackoverflow.com/questions/4964130/target-blank-vs-target-new